sdk/include/ddk/scsi.h

```diff
@@ -1239,11 +1239,11 @@ typedef union _CDB {
     UCHAR ExpectedSectorType:3;
     UCHAR Lun:3;
     _ANONYMOUS_UNION union {
-      struct _LBA {
+      struct {
```

Comment:
This is not what the native headers do and removing these tags can easily break other code that relies on them. It seems like GCC has some retardedness going on as usual, making up issues where there are none. Anyway we can work around this in a way that is compatible with the original definition:
Move these struct declarations to before the main struct declaration and replace this this with `struct _LBA LBA;`

Reply:
Thanks for the idea. You suggest moving the declaration of the `struct _LBA {...};` structure to the global scope. This is also not very good.